### DISCUSSION TITLE
Welcome to Ether-Chess-OTC Discussions!

### DISCUSSION THREAD

**Discussion by Navneet1206** [Navneet1206/Ether-Chess-OTC#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yashpandav** [comment]
Hello @Navneet1206 I’d greatly appreciate it if you could provide some insights into how this pull request will be counted in the SWOC competition. As this is my first time contributing, I want to ensure that my contributions are tracked accurately and align with the competition’s guidelines.

Once again, thank you for your time and support throughout this process! I’m really looking forward to working on another issue soon.

Best regards,
Yash Pandav
